format/location-filter.js

```diff
@@ -1,6 +1,14 @@
 const formatLocationFilter = (stops, addresses, poi) => {
-	if (stops && addresses && poi) return 'ALL'
-	return (stops ? 'S' : '') + (addresses ? 'A' : '') + (poi ? 'P' : '')
+	if (stops && addresses && poi) {
+		return 'ALL'
+	}
+	return (stops
```

Comment:
I guess this is not easy to make more readable while not breaking other section's formatting? 😄

Reply:
We could switch of the curly rule: <https://eslint.org/docs/latest/rules/curly>. I think it would be perfect fine if we don't force curly braces.

Then it could remain a one-liner, if that's what you mean by "more readable"(?) :slightly_smiling_face:

Reply:
No, I mean the (to me) unintuitive multi-line formatting of the ternary expressions.

For example, indent the "components" of a ternary expression (condition, truthy expression, falsy expression) intended, each on a separate line, and the wrapping parantheses one indentation less.

Reply:
You're talking about this part, right?

```js
	return (stops
		? 'S'
		: '') + (addresses
		? 'A'
		: '') + (poi
		? 'P'
		: '');
};
```

You are right the formatting was not very readable. I couldn't get a better version without a bit manual work. These two variants would be possible:

1. The new current state with this rule `"@stylistic/multiline-ternary": ["error", "always-multiline"]`:

```js
	return (
		(stops ? 'S' : '')
		+ (addresses ? 'A' : '')
		+ (poi ? 'P' : '')
	);
```

2. This would be possible with this rule `"@stylistic/multiline-ternary": ["error", "always"]`:

```js
	return (
		(stops
			? 'S'
			: '')
			+ (addresses
				? 'A'
				: '')
				+ (poi
					? 'P'
					: '')
	);
```

Reply:
Yes, definitely `always-multiline` :+1: